backend/src/backend/project.gleam

```diff
@@ -0,0 +1,14 @@
+import backend/db
+import backend/web
+import gleam/result
+import wisp
+
+pub fn create_project(
+  projectid: String,
+  userid: String,
```

Comment:
You should create a wrapper type for a project in Web, just like we have for User.